### ISSUE TITLE
Blob column causes import to fail and hangs sequel pro repeatedly

### ISSUE THREAD

**Issue by markl181** [sequelpro/sequelpro#3216]
I have a (fairly small) sql file that was exported from Sequel Pro a couple of months ago. I'm setting up a new machine and trying to re-import the same file back into Sequel Pro. When the import ran, I got an error and it failed at the table that contains a mediumblob column. When I attempt to open that table in the UI, Sequel Pro hangs (becomes unresponsive and has to be force quit). When I attempt to drop that table either from the UI or via query, Sequel Pro hangs. The same thing happens if I try to drop the database or any other table (although I can view the other tables). Please let me know how I can fix this. 

I'm not sure if it's a bug or not (except perhaps if a corrupt table due to the import is causing all this).

**Comment by dmoagx** [comment]
1. Try this build: https://sequelpro.com/builds/Sequel-Pro-Build-3477d22387.zip
2. If it still happens, open Activity Monitor, select "Sequel Pro (not responding)", go to View > Sample Process and post the results

**Comment by markl181** [comment]
Thanks for your response - unfortunately I needed to get this done so I had to resort to the command line. If it helps at all, I manually dropped each table from the command line and then reimported each table using the create table and insert statements from the sql export file. The fact that this worked suggests to me that there is some type of issue here. If I get a chance I’ll rerun the import to get the error again.
